### PR TITLE
fix: balance format precision with tooltips

### PR DESCRIPTION
**Pull request by helciofranco**
- Closes https://github.com/FuelLabs/fuels-wallet/issues/1564
- Closes https://github.com/FuelLabs/fuels-wallet/issues/1575

---

I've also removed the `balances` from our browser storage. Let's keep it fetching directly from the node.

| 📷  Tiny | Regular | Huge |
| --- | --- | --- |
| small amounts | not so small |huge amount |

### CODE REVIEW

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
this PR is changing too much, the issue aims to fix the balance only, but I see the Approve Transaction screen is showing balance with <0.00001. Revert the changes in places we don't want and keep them only for showing correct balances.

File: packages/app/src/systems/Account/components/BalanceAssets/BalanceAssets.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const BalanceAssets = ({
           amount={balance.amount}
           onRemove={onRemove}
           onEdit={onEdit}
-          showActions
+          showActions={false}
```
Comment (arthurgeron) [inline packages/app/src/systems/Account/components/BalanceAssets/BalanceAssets.tsx:48]: default is false

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
2 problems here
- its showing wrong balance for the unknown tokens, before it was with zero units and now it's using like 9 units
- tooltip is showing only 0

image

**Comment by helciofranco** [comment]
@LuizAsFight nice catch, fixes has been pushed.

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**